nx-X11/programs/Xserver/Imakefile

```diff
@@ -361,6 +363,7 @@ NXAGENTNXLIBS = -L../../../nxcomp/src/.libs  \
 #else
 NXAGENTNXLIBS = -L../../../nxcomp/src/.libs  \
                 -L../../../nxcompshad/src/.libs \
+                -L../../lib/src/.libs \
```

Comment:
the -L lines seems identical for most, if not all, cases.  So can't we make that simpler?